global_helpers/panther_oss_helpers.py

```diff
@@ -235,6 +237,32 @@ def set_key_expiration(key: str, epoch_seconds: int) -> None:
     )
 
 
+def put_dictionary(key: str, val: dict, epoch_seconds: int = None):
+    # Convert the data dictionary to a JSON string
+    data = json.dumps(val)
```

Comment:
I feel like we should error handle when `val` can't be `json.dumps()`'ed 

```pycon
>>> from datetime import datetime
>>> herps = {"thing": 1, "when": datetime.now()}
>>> import json
>>> json.dumps(herps)
```

Reply:
Added!